extensions/csrc/cuda/scaled_masked_softmax_kernel.cu

```diff
@@ -9,16 +9,462 @@
 #include <cuda_runtime.h>
```

Comment:
Was this file committed by 1) git mv (from `extensions/csrc/cuda/scaled_masked_softmax.h`) and 2) editing and commit ?

Reply:
The second way